pkg/api/owners.go

```diff
@@ -0,0 +1,687 @@
+}
+
+func (f *FieldOwners) ClaimCPURealtimeRuntime(plugin string) error {
+	return f.ClaimSimple(Field_CPURealtimeRuntime.Key(), plugin)
+}
+
+func (f *FieldOwners) ClaimCPURealtimePeriod(plugin string) error {
+	return f.ClaimSimple(Field_CPURealtimePeriod.Key(), plugin)
+}
+
+func (f *FieldOwners) ClaimCPUSetCPUs(plugin string) error {
+	return f.ClaimSimple(Field_CPUSetCPUs.Key(), plugin)
+}
+
+func (f *FieldOwners) ClaimCPUSetMems(plugin string) error {
+	return f.ClaimSimple(Field_CPUSetMems.Key(), plugin)
+}
+
+func (f *FieldOwners) ClaimPidsLimit(plugin string) error {
+	return f.ClaimSimple(Field_PidsLimit.Key(), plugin)
+}
+
+func (f *FieldOwners) ClaimHugepageLimit(size, plugin string) error {
+	return f.ClaimCompound(Field_HugepageLimits.Key(), size, plugin)
+}
+
+func (f *FieldOwners) ClaimBlockioClass(plugin string) error {
+	return f.ClaimSimple(Field_BlockioClass.Key(), plugin)
+}
+
+func (f *FieldOwners) ClaimRdtClass(plugin string) error {
+	return f.ClaimSimple(Field_RdtClass.Key(), plugin)
+}
+
+func (f *FieldOwners) ClaimCgroupsUnified(key, plugin string) error {
+	return f.ClaimCompound(Field_CgroupsUnified.Key(), key, plugin)
+}
+
+func (f *FieldOwners) ClaimCgroupsPath(plugin string) error {
+	return f.ClaimSimple(Field_CgroupsPath.Key(), plugin)
+}
+
+func (f *FieldOwners) ClaimOomScoreAdj(plugin string) error {
+	return f.ClaimSimple(Field_OomScoreAdj.Key(), plugin)
+}
+
+func (f *FieldOwners) ClaimRlimit(typ, plugin string) error {
+	return f.ClaimCompound(Field_Rlimits.Key(), typ, plugin)
+}
+
+func (f *FieldOwners) ClearCompound(field int32, key, plugin string) {
+	m, ok := f.Compound[field]
+	if !ok {
+		m = NewCompoundFieldOwners()
+		f.Compound[field] = m
+	}
+
+	m.Owners[key] = MarkForRemoval(plugin)
+}
+
+func (f *FieldOwners) ClearSimple(field int32, plugin string) {
+	f.Simple[field] = MarkForRemoval(plugin)
+}
+
+func (f *FieldOwners) ClearAnnotation(key, plugin string) {
+	f.ClearCompound(Field_Annotations.Key(), key, plugin)
+}
+
+func (f *FieldOwners) ClearMount(destination, plugin string) {
+	f.ClearCompound(Field_Mounts.Key(), destination, plugin)
+}
+
+func (f *FieldOwners) ClearDevice(path, plugin string) {
+	f.ClearCompound(Field_Devices.Key(), path, plugin)
+}
+
+func (f *FieldOwners) ClearEnv(name, plugin string) {
+	f.ClearCompound(Field_Env.Key(), name, plugin)
+}
+
+func (f *FieldOwners) ClearArgs(plugin string) {
+	f.ClearSimple(Field_Args.Key(), plugin)
+}
+
+func (f *FieldOwners) Conflict(field int32, plugin, other string, qualifiers ...string) error {
+	return fmt.Errorf("plugins %q and %q both tried to set %s",
+		plugin, other, qualify(field, qualifiers...))
+}
+
+func (f *FieldOwners) CompoundOwner(field int32, key string) (string, bool) {
+	if f == nil {
+		return "", false
+	}
+
+	m, ok := f.Compound[field]
+	if !ok {
+		return "", false
+	}
+
+	plugin, ok := m.Owners[key]
+	return plugin, ok
+}
+
+func (f *FieldOwners) SimpleOwner(field int32) (string, bool) {
```

Comment:
This doesn't appear to be called outside `owners.go` or even outside of `FieldOwners` methods.  Can it be made private?

Reply:
Made it private.